yarn-error.log

```diff
@@ -0,0 +1,199 @@
+Arguments: 
```

Comment:
we don't want to save this file, so it can be deleted. I'll add it to the gitignore for the future